default-recommendations/conditional-shortcuts.rkt

```diff
@@ -268,13 +268,28 @@
   (when condition body))
 
 
+(define-refactoring-rule explicit-cond-else-void
+  #:description "Add an explicit `[else (void)]` clause to make the default behavior clear."
+  #:literals (cond else void)
+  (cond-id:cond clause ...)
+  #:when (syntax-property this-syntax 'expression-result)
```

Comment:
You can remove this `#:when` condition, if the property is false then the `(equal? ... 'ignored)` condition on the next line will be false.